app/package.json

```diff
@@ -26,7 +26,7 @@
     "build:script": "parcel build src/script.js --out-dir ../dist/ --no-cache",
     "watch:script": "parcel watch src/script.js --out-dir ../dist/ --no-hmr",
     "devserver": "parcel serve index.html -p 8001 --out-dir ../dist/ --no-cache",
-    "start": "npm run sync-assets && npm run watch:script & npm run devserver",
+    "watch": "npm run sync-assets && npm run watch:script & npm run devserver",
```

Comment:
`start` is nice because it's a shortcut with `npm start`

Reply:
We changed it to "watch" because the plugin's front end building mechanism relies on an npm script named "watch" in app/. We found this to be much faster than running a watcher ourselves, but not sure if this will hold.

On the meantime, we could:
A) Have the plugin look for a "start" script instead.
B) Have both, with "start" just calling "npm run watch".

Reply:
B) sounds very reasonable; the plugin can also bail or warn if the app doesn't have a `watch` script.

We should definitely not be running a watcher ourselves to re-compile JS; that's something the app's build system should handle.